pkg/backup/backup_pv_action.go

```diff
@@ -55,8 +54,12 @@ func (a *backupPVAction) Execute(item runtime.Unstructured, backup *v1.Backup) (
 	pvc := item.UnstructuredContent()
 
 	volumeName, err := collections.GetString(pvc, "spec.volumeName")
```

Comment:
Consider using `collections.Exists()`?

Reply:
I didn't change it because we still use the volumeName if it's there, and under the hood GetString and Exists go through the same codepath anyway.

Reply:
Ok. Hmm looks like as of kube 1.9 they've finally exported the unstructured helper funcs, so we can maybe get rid of our collections package for the most part.